internal/csi-addons/rbd/replication.go

```diff
@@ -649,6 +652,28 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 	return resp, nil
 }
 
+func getError(err error) error {
+	errorStatusMap := map[error]codes.Code{
+		corerbd.ErrLocalStateNotFound:          codes.Internal,
+		corerbd.ErrInvalidArgument:             codes.InvalidArgument,
+		corerbd.ErrDisableImageMirroringFailed: codes.Internal,
+		corerbd.ErrMirroringInfoNotFound:       codes.Internal,
+		corerbd.ErrAborted:                     codes.Aborted,
+		corerbd.ErrFailedPrecondition:          codes.FailedPrecondition,
+		corerbd.ErrResyncImageFailed:           codes.Internal,
+		corerbd.ErrServiceUnavailable:          codes.Unavailable,
```

Comment:
```suggestion
		corerbd.ErrLocalStateNotFound:          codes.Internal,
		corerbd.ErrResyncImageFailed:           codes.Internal,
		corerbd.ErrDisableImageMirroringFailed: codes.Internal,
		corerbd.ErrMirroringInfoNotFound:       codes.Internal,
		corerbd.ErrInvalidArgument:             codes.InvalidArgument,
		corerbd.ErrAborted:                     codes.Aborted,
		corerbd.ErrFailedPrecondition:          codes.FailedPrecondition,
		corerbd.ErrServiceUnavailable:          codes.Unavailable,
```
nit: better to have it ordered.